modules/nf-core/polypolish/polish/main.nf

```diff
@@ -0,0 +1,50 @@
+process POLYPOLISH {
+    tag "$meta.id"
+    label 'process_high_memory'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/polypolish:0.6.0--hdbdd923_0':
+        'biocontainers/polypolish:0.6.0' }"
+
+    input:
+    // input:  polypolish polish [OPTIONS] <ASSEMBLY> [SAM]
+    tuple val(meta), path(fasta)
+    tuple val(meta), path(sam)
+
+    output:
+    // polypolish polish draft.fasta filtered_1.sam filtered_2.sam > polished.fasta
+    tuple val(meta), path("*.fasta"), emit: fasta
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    polypolish \\
+        polish \\
+        $args \\
+        ${fasta} \\
+        $sam > ${prefix}_polished.fasta
```

Comment:
```suggestion
        ${sam} > ${prefix}.fasta
```

As we can't assume what a pipeline developer wants to name their files. You'll need to add an extra check in the variable bit of the script block [^1]

[^1]: https://nf-co.re/docs/guidelines/components/modules#command-file-output-naming